src/Twig/Pagination.php

```diff
@@ -20,6 +20,20 @@ class Pagination {
 	 */
 	private array $page_cache = [];
 
+	/**
+	 * WP Query.
+	 *
+	 * @var WP_Query
+	 */
+	private WP_Query $query;
```

Comment:
This property definition may be omitted when using constructor property promotion. Like so:

```php
	public function __construct( private WP_Query $wp_query ) {
```